src/config.ts

```diff
@@ -1222,4 +1231,23 @@ export default {
    * there can be in that history.
    */
   BUFFERED_HISTORY_MAXIMUM_ENTRIES: 200,
+
+  /**
+   * Minimum buffer (in seconds ) we should have, regardless of memory
+   * constraints
+   */
+  MIN_BUFFER_LENGTH : 5,
+
+  /**
+   * Minimum distance (in seconds) from the end
+   * we should be able to download
+   * Before starting to garbage collect previous segments
+   */
+  MIN_BUFFER_DISTANCE_BEFORE_CLEAN_UP: 10,
+
+  /**
+   * How much close to the current position we should garbage collect
+   * If we have the memory is saturated
+   */
+  UPTO_CURRENT_POSITION_CLEANUP : 5,
```

Comment:
This seems to be only for the buffer behind. I found the description unclear about that, especially on its difference with `MIN_BUFFER_LENGTH`

Reply:
Changed